src/main/java/se/kth/iv1201/grupp13/recruiterapplication/repository/UserRepository.java

```diff
@@ -17,10 +17,37 @@
 @Repository
 @Transactional(propagation = Propagation.MANDATORY) // Support current transaction, throw an exception if there is no transaction currently.
 public interface UserRepository extends JpaRepository<User, Long> {
-
+    /**
+     * Returns the user with the specified user name and password.
+     *
+     * @param username The user name to search for.
+     * @param password The password to search for.
+     * @return A user with the specified user name and password.
+     */
     User findByUsernameAndPassword(String username, String password);
-    List<User>  findByNameLike(String name);
-    List<User> findByAvailabilities(List<Availability> availabilities);    
+
+    /**
+     * Returns the users with the specified name.
+     *
+     * @param name The user's name to search for.
+     * @return A list containing all users with the specified name.
+     */
+    List<User> findByNameLike(String name);
+    
+    /**
+     * Returns the users with the specified availabilities.
+     *
+     * @param availabilities The availabilities to search for.
+     * @return A list containing all users with the specified availabilities.
+     */    
+    List<User> findByAvailabilities(List<Availability> availabilities); 
```

Comment:
This may work, but I'm not sure. We should run some tests for this as soon as we got the database up and running (we will soon).